### DISCUSSION TITLE
我的RSS订阅 | Goalonez Blog

### DISCUSSION THREAD

**Discussion by Goalonez** [Goalonez/goalonez.github.io#131] [Announcements]
https://blog.goalonez.site/blog/%E6%88%91%E7%9A%84RSS%E8%AE%A2%E9%98%85.html 

 万变不离其宗

**Comment by HalbertZ** [comment]
感谢分享，下载了 NetNewsWire，添加你列表中部分 RSS 时会提示Feed not found。想问问是什么原因？

**Comment by Goalonez** [comment]
> @HalbertZ
> 感谢分享，下载了 NetNewsWire，添加你列表中部分 RSS 时会提示Feed not found。想问问是什么原因？

方便说下具体是哪一个吗

**Comment by SuperXCR** [comment]
我使用freshrss，很多rsshub添加不上

**Comment by caoyirui** [comment]
竹新社跟联合早报相比哪个更中立？

**Comment by Goalonez** [comment]
> @caoyirui
> 竹新社跟联合早报相比哪个更中立？

不太清楚，没怎么看。😅

**Comment by Sunyin0818** [comment]
Follow（现在改名叫Folo了）看RSS有些站（e.g. V2EX）的评论加载不出来，查看源格式的话评论的样式也很丑，有点难受。只关注文章主体的话，还是不错的

**Comment by Goalonez** [comment]
> @Sunyin0818
> Follow（现在改名叫Folo了）看RSS有些站（e.g. V2EX）的评论加载不出来，查看源格式的话评论的样式也很丑，有点难受。只关注文章主体的话，还是不错的

是的，现在改叫folo了。我目前主要靠推送到tg来看rss内容，其他客户端很少用了。
